ext/mysqlnd/mysqlnd_vio.c

```diff
@@ -414,6 +414,29 @@ MYSQLND_METHOD(mysqlnd_vio, set_client_option)(MYSQLND_VIO * const net, enum_mys
 			net->data->options.ssl_verify_peer = val;
 			break;
 		}
+		case MYSQL_OPT_TLS_VERSION:
+			if (strcmp(value, "TLSv1") == 0) {
```

Comment:
Actually that entire function seems to be duplicated between mysqlnd_vio.c and mysqlnd_net.c. It seems that the _net version is no longer used, but I wasn't sure, so I just copied was what there.